src/components/EditProfileModal/EditProfileModal.css

```diff
@@ -0,0 +1,84 @@
+.edit-profile {
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.7);
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+.edit-profile-card {
+  min-width: 15rem;
+  width: 40%;
+  position: absolute;
+  background-color: var(--secondary-bg-color);
+  padding: 0.5rem 0.5rem 0rem;
+  border: 3px solid var(--primary-color);
+  border-radius: 5px;
+  top: 40%;
+  left: 50%;
+  transform: translateX(-50%) translateY(-40%);
```

Comment:
translate(-50%,-50%) is the shorthand for translateX and translateY

Reply:
I will Change.